boba-docs/docs/basics/6_deploy-contract.md

```diff
@@ -179,7 +185,27 @@ Summary
 ✨  Done in 10.11s.
 ```
 
-That's pretty much it. Contracts deployed! Tutorial complete. Hopefully now you know the basics of working with Boba!
+## Verify the Contract
+
+Using our tool [Bobascan](https://testnet.bobascan.com/), you can verify the contract has been deployed and is valid. Sign in by connecting your crypto wallet to Bobascan, then navigate to your transactions. [TODO: ADD IMAGES OF WHERE TO FIND TRANSACTIONS AND CONTRACT ADDRESS] This will lead you to the contract address of your newly deployed contract. Copy this address, then replace "`YOUR_CONTRACT_ADDRESS_HERE`" with it in the command below to verify your contract:
+
+```bash
```

Comment:
We have yarn or npx toggles elsewhere.  Shall we add this to the package.json so we can do both here too?